build.sbt

```diff
@@ -22,7 +24,7 @@ supportNumberFormats := false
 
 libraryDependencies ++= Seq(
   "io.github.cquiroz" %%% "scala-java-time" % "2.3.0",
-  "org.portable-scala" %% "portable-scala-reflect" % "1.1.1"
+  ("org.portable-scala" %%% "portable-scala-reflect" % "1.1.1").cross(CrossVersion.for3Use2_13)
```

Comment:
See https://github.com/portable-scala/portable-scala-reflect/issues/23, unfortunately no Scala 3 release is coming.